src/NUnitEngine/nunit.engine/RuntimeFramework.cs

```diff
@@ -53,6 +53,19 @@ public enum RuntimeType
     }
 
     /// <summary>
+    /// Enumeration identifying profile of .NET Framework
+    /// </summary>
+    public enum Profile
+    {
+        /// <summary>Profile Unspecified</summary>
+        Unspecified,
+        /// <summary>Client Profile</summary>
+        Client,
+        /// <summary>Full Profile</summary>
+        Full
+    }
```

Comment:
My logic with enum/unspecified was that it seemed wrong to specify all the non-.NET frameworks as 'full profile' when there wasn't such a concept. Can change that if you think it's best kept simple however!